app/views/govuk_publishing_components/component_guide/component_doc/_preview.html.erb

```diff
@@ -1,3 +1,4 @@
-<div data-module="test-a11y" class="component-guide-preview <% if example.right_to_left? %>direction-rtl<% end %>">
-  <%= render component_doc.partial_path, example.html_safe_data %>
+<div class="js-test-a11y" data-module="test-a11y">
+  <%= render "govuk_publishing_components/component_guide/component_doc/component", component_doc: @component_doc, example: example, preview_page: false %>
+  <div class="component-guide-preview component-guide-preview--warning" data-module="test-a11y-warning" data-content="Potential Accessibility Issues: Need Manual Check"></div>
```

Comment:
could we do 'aria-label' instead of 'data-content'?

Reply:
@nickcolley I did look into this - the data-content title is read out without using an aria-label. If you use an aria-label, it actually gets read out twice, which didn't seem good?

Reply:
Ah so since it was in the pseudo element it was read out, fair enough 👍